src/components/Logo/Logo.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LogoWrapper } from './Logo.styles';
+
+const Logo = ({ width, height }) => {
+  return <LogoWrapper width={width} height={height} />;
+};
+
+Logo.propTypes = {
+  width: PropTypes.number,
+  height: PropTypes.number
+};
```

Comment:
You don't need hegith and width as props, define in styles.

Reply:
I have removed height and width props and hardcoded values in styles.